assets/js/components/edit-product-link/index.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Icon, external } from '@woocommerce/icons';
+import { ADMIN_URL } from '@woocommerce/settings';
+import { InspectorControls } from '@wordpress/block-editor';
+
+/**
+ * Component to render an edit product link in the sidebar.
+ */
+const EditProductLink = ( { productId } ) => {
```

Comment:
I see this is not new on this branch - I'm assuming it's been reviewed elsewhere and there's nothing new/changed in this branch. Code looks fine though :)

Reply:
Yeah just moved so it can be consumed by multiple blocks.